view/theme/frio/php/standard.php

```diff
@@ -22,6 +22,9 @@
 
 use Friendica\DI;
 
+$baseurl = DI::baseUrl();
```

Comment:
When we assign the `$baseurl` variable, why aren't we using it in line 35?

Reply:
You're right, this variable isn't used anywhere.

Reply:
What are your plans about this variable? I'm confused that it had been added here in this PR :-)

Reply:
I’ll remove it. It was not added but moved from its location inside the `<head/>` tag.

Reply:
Done.